config/locales/api/am_ET.yml

```diff
@@ -12,11 +12,11 @@ am-ET:
       expired_otp: "OTPዎ ጊዜው አልፏል። አዲስ ይጠይቁ።"
       invalid_password: "የይለፍ ቃልዎ አይዛመድም። ድጋሚ ይሞክሩ?"
       account_locked: "መለያዎ ለሚቀጥሉት %{minutes} ደቂቃዎች ተቆልፏል። እባክዎ ይቆዩ እና እንደገና ይሞክሩ።"
-  sms:
+  communications:
```

Comment:
Hopefully this change doesn't cause confusion. There are no actual communication records for OTP  requests, but SMS is pretty obviously wrong, since it contains whatsapp translations and will probably also end up with Imo translations.